### DISCUSSION TITLE
Dependency error in shopify hydrogen react node

### DISCUSSION THREAD

**Discussion by developerparisgroup** [community/community#101404] [Actions]
### Select Topic Area

Bug

### Body

The engine "node" is incompatible with this module. Expected version ">=18.0.0". Got "16.20.2"
error Found incompatible module.

**Comment by brenoepics** [comment]
Looks like your node js version is too old, try to update it.
https://nodejs.org/en/download/current
tutorials:
https://blog.hubspot.com/website/update-node-js
https://mcengkuru.medium.com/how-to-update-node-js-to-any-version-a-step-by-step-guide-d4ce747ac50f
